.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- Specify the issue ID that this pullrequest is solving -->
+Fix #XXX
+
+<!-- Describe the changes introduced by this pull request -->
+
+## Test
+
+To test this pull request, you can run the following commands:
+
+```shell
+cp <to_package_directory>
+go test
```

Comment:
should it be `go test ./...` so that all the nested directories are tested?

Reply:
That won't work, because you need a cluster and the fleet-examples repo for some tests. The unit tests provide ~4% coverage at the moment. They don't help with verifying a PR. 

I'd remove the testing part from the template. We only have the workflows at the moment.

We could mention `golangci-lint run`.

Reply:
I can propose instead 

<!-- Please describe, the way to test this pull request -->